cli/kata-env_test.go

```diff
@@ -30,8 +30,9 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-const testProxyVersion = "proxy version 0.1"
-const testShimVersion = "shim version 0.1"
+var testProxyVersion = VersionInfo{Version: "proxy version 0.1", Major: "0", Minor: "1"}
```

Comment:
Great to see some new tests. It would be great though if you could test a larger number of scenarios:

- runtime older than proxy or shim
- runtime older than proxy and shim
- runtime newer than proxy or shim
- runtime newer than proxy and shim
- *etc*

I think the easiest way to add in a table-based set of semi-exhaustive tests would be to add in that strict option to allow you to test for `error` in `checkVersionConsistencyInComponents()`.